components/modal/style/confirmCmp.tsx

```diff
@@ -53,6 +53,9 @@ const genModalConfirmStyle: GenerateStyle<ModalToken> = (token) => {
         flexDirection: 'column',
         flex: 'auto',
         rowGap: token.marginXS,
+      },
+
+      [`${confirmComponentCls}-paragraph:nth-child(2)`]: {
```

Comment:
`:nth-child(2)` 这种写法并不稳定。

Reply:
He is Korean and may need to communicate in English.

Reply:
@afc163 you're right.
Icon of modal can be many element. (by Fragment)
The solution that the selector changes `:nth-child(2)` to `:nth-last-child`.
Because, The elements in modal body is sorted `icon element` ~ `paragraph element`, all of situation paragraph element will be last element.

Let's fix it like that first.